Liquid_SF_1.tex

```diff
@@ -0,0 +1,271 @@
+    \item \daqfull{}      \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \controlfull{}  \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end sign off -- }}}
+
+\newpage
+% begin prior to start section -- {{{
+\subsection{Prior to Start}
+\begin{checklist}
+    \item Ensure that all personnel as defined above are available and have completed the sign-off.
+    \item Ensure that the following personnel have walkie-talkies and communication is functional:
+    \begin{checklist}
+        \item \ops{}
+        \item \secondary{}
+        \item \daq{}
+        \item \control{}
+    \end{checklist}
+    \item Ensure that all spectators and test personnel are wearing safety glasses and hearing protection.
+    \item Ensure that \primary{} and \secondary{} are wearing face shields and have no exposed skin.
+    \item Ensure that \primary{} is wearing thermal gloves.
+    \item Ensure that \secondary{} is in possession of a multimeter.
+    \item Ensure that \ops{} is in possession of the system control key.
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end prior to start section -- }}}
+
+\newpage
+
+% begin Remote Control Procedure section -- {{{
+\subsection{Static Fire Test - Remote Control Procedure}
+\begin{checklist}
+    \item \secondary{}: Confirm that the ignition wires are not connected to the engine.
+    \item \primary{}: Confirm that the following valves are initially closed:
+    \begin{checklist}
+        \item SC-1 (Oxidizer Supply Valve)
+        \item TK-1 (Pressurant Supply Valve)
+        \item BA-1 (Oxidizer Pressurant Shutoff Valve)
+        \item BA-2 (Fuel Pressurant Shutoff Valve)
+        \item BA-3 (Oxidizer Parallel Fill Valve)
+        \item BA-5 (Oxidizer Fill Line Vent Valve)
+        \item BA-8 (Oxidizer Tank Dump Valve)
+        \item BA-9 (Fuel Tank Vent Valve)
+        \item BA-10 (Pressurant Line Vent Valve)
+        \item MV-1 (Oxidizer Motorized Fill Valve)
+        \item MV-2 (Pressurant Remote Valve)
+        \item MV-3 (Oxidizer Motorized Vent Valve)
+        \item MV-4 (Fuel Injector Valve)
+        \item IJ-1 (Oxidizer Injector Valve)
+    \end{checklist}
+    \item \primary{}: Confirm that the following valves are initially open:
+    \begin{checklist}
+        \item BA-4 (Oxidizer Series Fill Valve)
+        \item BA-6 (Oxidizer Shutoff Valve)
+        \item BA-7 (Fuel Shutoff Valve)
+    \end{checklist}
+    \item \primary{}: Confirm that CV-1 is adjusted to the lowest pressure setting.
+    \item \daq{}: Confirm that all pressure transducers are reading atmospheric pressure.
+    \item \daq{}: Confirm that all load cells are reading the determined zero point.
+    \item \daq{}: Confirm that all thermistors are reading ambient temperature.
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \secondary{}: Confirm that no personnel are present in the testing area other than \primary{} and \secondary{}.
```

Comment:
There's no mention of perimeter in these ops, it's entirely possible that I'm out of the loop. Do we no longer have people at the perimeter stopping people from walking towards the engine? Are the postings enough?